gqlgen/graph/schema.graphqls

```diff
@@ -3,7 +3,18 @@
 # https://gqlgen.com/getting-started/
 
 type Quote {
-  id: ID!
+  #id of quote
+  id: String!
+  #quote body
   quote: String!
+  #author of quote
```

Comment:
Love the comments. When we bring in a linter (code formatting/checking tool), it will force you to add spaces between the # and comment, or you could just fix now. Up to you!
```suggestion
  # id of quote
  id: String!
  # quote body
  quote: String!
  # author of quote
```